routers/comments.py

```
@@ -0,0 +1,75 @@
+        )
+    )
+    return CommonResponse(
+        status = "success",
+        message= "성공적으로 댓글 목록을 불러왔습니다.",
+        data = paginated_data
+    )
+
+
+@router.patch("/{comment_id}", response_model=CommonResponse[CommentResponse],status_code=status.HTTP_200_OK)
+async def update_comment(
+        post_id: int,  # prefix에서 주입됨
```

Ruff (0.14.13)

Unused function argument: `post_id`

(ARG001)